src/pages/index.js

```diff
@@ -1,20 +1,74 @@
 import Head from "next/head";
-import styles from "../styles/Home.module.css";
+import { useState, useEffect, useCallback } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Cookies from "universal-cookie";
+
+import axios from "../lib/axios";
+import Navbar from "../components/Navbar/Navbar";
+import TodoForm from "../components/TodoForm/TodoForm";
+import { Container } from "@material-ui/core";
+import TodoLists from "../components/TodoLists/TodoLists";
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+        marginTop: theme.spacing(4),
+    },
+}));
+
+export default function Dashboard() {
+    const classes = useStyles();
+    const cookies = new Cookies();
+    const [email, setEmail] = useState("");
+    const [todos, setTodos] = useState([]);
+
+    const getTodos = () => {
+        axios
+            .get("/todos")
+            .then((res) => setTodos(res.data))
+            .catch((err) => console.log(err));
+    };
+
+    const addTodo = useCallback((todo, event) => {
+        event.preventDefault();
+        console.log(todo);
+    }, []);
+
+    const updateTodo = useCallback((todo, event) => {
+        event.preventDefault();
+        console.log(todo);
+    }, []);
+
+    const removeTodo = useCallback((todo, event) => {
+        event.preventDefault();
+        console.log(todo);
+    }, []);
```

Comment:
After implement this you can remove the logger yaaa, and use snackbar for error handling